lib/sourcemap.js

```diff
@@ -82,12 +82,22 @@ async function SourceMap(options) {
     }
 
     function add(source, gen_line, gen_col, orig_line, orig_col, name) {
+        let generatedPos = { line: gen_line, column: gen_col }
```

Comment:
Missing trailing semicolon.

Reply:
Seems like a lint rule could enforce this. Fixed up.